pkg/abi/linux/fuse.go

```diff
@@ -252,3 +252,23 @@ type FUSEOpenOut struct {
 
 	padding uint32
 }
+
+// FUSEReleaseIn is the request sent by the kernel to the daemon,
+// to negotiate flags and get file handle.
```

Comment:
This looks somehow confusing, should we consider clarifying this? (https://libfuse.github.io/doxygen/structfuse__operations.html#abac8718cdfc1ee273a44831a27393419)